[Android] CHIPTool crashes when network is not available

#### Problem

 * actual behavior
Android app crashes, when network is not reachable:

```
2020-11-12 09:21:45.252 16696-16696/com.google.chip.chiptool E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.google.chip.chiptool, PID: 16696
    java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:448)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
     Caused by: java.lang.reflect.InvocationTargetException
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807) 
     Caused by: chip.devicecontroller.ChipDeviceControllerException: OS Error 2101 (0x00000835): Network is unreachable
        at chip.devicecontroller.ChipDeviceController.beginSendCommand(Native Method)
        at chip.devicecontroller.ChipDeviceController.beginSendCommand(ChipDeviceController.java:85)
        at com.google.chip.chiptool.clusterclient.OnOffClientFragment.sendCommand(OnOffClientFragment.kt:120)
        at com.google.chip.chiptool.clusterclient.OnOffClientFragment.access$sendCommand(OnOffClientFragment.kt:19)
```

 * expected behavior
App does not crash. Probably show a toast that Network is unreachable.

 #### Proposed Solution

Catch `chip.devicecontroller.ChipDeviceControllerException` on all `beginSendCommand()` calls.

--- a/src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/OnOffClientFragment.kt
+++ b/src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/OnOffClientFragment.kt
@@ -8,6 +8,7 @@ import android.view.ViewGroup
 import androidx.fragment.app.Fragment
 import chip.devicecontroller.ChipCommandType
 import chip.devicecontroller.ChipDeviceController
+import chip.devicecontroller.ChipDeviceControllerException
 import com.google.chip.chiptool.ChipClient
 import com.google.chip.chiptool.GenericChipDeviceListener
 import com.google.chip.chiptool.R
@@ -93,7 +94,11 @@ class OnOffClientFragment : Fragment() {
     commandStatusTv.text =
         requireContext().getString(R.string.send_command_type_label_text, chipCommandType.name)
 
-    deviceController.beginSendCommand(commandType)
+    try {
+      deviceController.beginSendCommand(commandType)
+    } catch (e: ChipDeviceControllerException) {
+      commandStatusTv.text = e.toString()
+    }
   }
 
   companion object {

